include/jwt-cpp/jwt.h

```diff
@@ -967,12 +967,28 @@ namespace jwt {
 				size_t len = EVP_PKEY_size(pkey.get());
 				std::string res(len, '\0');
 
+				// LibreSSL is the special kid in the block, as it does not support EVP_DigestSign.
+				// OpenSSL on the otherhand does not support using EVP_DigestSignUpdate for eddsa, which is why we end up with this mess.
+				#ifdef LIBRESSL_VERSION_NUMBER
+				ERR_clear_error();
+				auto e = EVP_DigestSignUpdate(ctx.get(), reinterpret_cast<const unsigned char*>(data.data()), data.size());
+				if(e != 1) {
```

Comment:
To keep it consistent

```suggestion
				if(EVP_DigestSignUpdate(ctx.get(), reinterpret_cast<const unsigned char*>(data.data()), data.size()) != 1) {
```